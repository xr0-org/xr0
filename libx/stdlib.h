#ifndef STDLIB_H
#define STDLIB_H

axiom void *
malloc(int size) [ .alloc result; ];

axiom void
free(void *ptr) [ .dealloc ptr; ];

#endif
