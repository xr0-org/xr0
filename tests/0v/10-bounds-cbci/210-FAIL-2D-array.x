#include <stdlib.h>

int
foo()
{
	char c;
	char *a[4][3];
	a[4][0] = &c;
}
