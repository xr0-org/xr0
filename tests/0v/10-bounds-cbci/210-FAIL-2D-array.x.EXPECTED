10-bounds-cbci/210-FAIL-2D-array.x:8:14: out of bounds (foo)
